### PR TITLE
User timeout front end shared code

### PR DESCRIPTION
**Pull request by richmilton**
## Ticket
[2.2.1 Timing Adjustable - Time Limit Present On Page](https://vmddefra.atlassian.net/browse/DEV-11244)

## Overview
Add js & css for accessible timeout feature

### CODE REVIEW

**Review by meyumeapps — APPROVED** [review]
LGTM!

**Review by bb128b — APPROVED** [review]

**Review by simon-technocloud — APPROVED** [review]
LGTM...only comment would be that I dont think you need it in a 'front-end' directory as the whole library is a specific hapi version....

File: front-end/govuk-timeout/js/govuk-timeout-warning.js [hunk 1]
```diff
@@ -0,0 +1,577 @@
...
+    if (window.NodeList.prototype.forEach) {
+      return nodes.forEach(callback);
+    }
+    for (let i = 0; i < nodes.length; i++) {
+      callback.call(window, nodes[i], i, nodes);
+    }
+  }
+
+  function TimeoutWarning($module) {
+    console.log($module)
```
Comment (neilD2) [inline front-end/govuk-timeout/js/govuk-timeout-warning.js:31]: did we want this in?

File: front-end/govuk-timeout/js/govuk-timeout-warning.js [hunk 2]
```diff
@@ -0,0 +1,577 @@
...
+
+    // Start watching for idleness
+    this.countIdleTime();
+
+    this.$closeButton.addEventListener('click', this.closeDialog.bind(this));
+    this.$module.addEventListener('keydown', this.escClose.bind(this));
+
+    // Debugging tip: This event doesn't kick in in Chrome if you have Inspector panel open and have clicked on it
+    // as it is now the active element. Click on the window to make it active before moving to another tab.
+    console.log('hello')
```
Comment (neilD2) [inline front-end/govuk-timeout/js/govuk-timeout-warning.js:88]: and this

**Review by neilD2 — APPROVED** [review]
LGTM

**Pull request merged**